ci/container_index/engine.py

```diff
@@ -1,11 +1,15 @@
+from os import mkdir, rmdir, path
```

Comment:
Add docs at the top of engine.py about it.

Reply:
Docs added